java/src/org/openqa/selenium/HasDownloads.java

```diff
@@ -27,7 +27,13 @@ public interface HasDownloads {
   /**
    * Requires downloads to be enabled.
    *
-   * <p>TODO: Create an example in the documentation and provide a link to it.
+   * <p>Example usage:
+   *
+   * <pre>{@code
+   * Capabilities capabilities = new Capabilities();
```

Comment:
Capabilities is an interface. I suggest used ChromeOptions as an implemented class in the example.

Reply:
@pujagani please check